src/core_plugins/timelion/public/directives/partials/suggestion.html

```diff
@@ -5,7 +5,7 @@
   <div
     class="suggestion"
     data-suggestion-list-item
-    ng-click="completeExpression($index)"
+    ng-click="onClickSuggestion({ suggestionIndex: $index })"
```

Comment:
why ? this calls onClickSuggestion which just calls completeExpression ...
i don't see any benefits, but additional code to read and nested function call ?